scripts/index.js

```diff
@@ -0,0 +1,99 @@
+const state = {
+    currentTemp: parseInt(document.getElementById("temp_val").textContent)
+} 
+
+const updateCityName = () => {
+    const inputName = document.getElementById("city_input").value;
+    const headerCity = document.getElementById("city_name");
+    headerCity.textContent = inputName;
+}
+
+const resetCityName = () => {
+    const headerCity = document.getElementById("city_name");
+    const inputText = document.getElementById("city_input");
+    headerCity.textContent = "Seattle";
+    inputText.value = "Seattle"
+}
+
+const selectSky = () => {
+    const skyChoice = document.getElementById("sky_choice").value;
+    let gardenBg = document.getElementById("garden_bg");
+    let weather = document.getElementById("skyemoji")
+    let color ="";
+    let sky = "";
+    if (skyChoice === "Rainy") {
+        color = "rainy";
+        sky = "🌧🌈⛈🌧💧⛈🌦🌧💧🌧🌈🌧";
+    } else if (skyChoice === "Sunny") {
+        color = "sunny";
+        sky = "☁️ ☀️ ☁️ ☀️ ☁️ ☀️ ☁️ ☀️ ☁️";
+    } else if (skyChoice === "Snowy") {
+        color = "snowy";
+        sky = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨";
+    } else if (skyChoice === "Cloudy") {
+        color = "cloudy";
+        sky = "☁️☁️ 🌤☁️ ☁️☁️ ☁️ 🌤 ☁️☁️";
+    }
+    gardenBg.classList = `${color}`;
+    weather.textContent = sky;
+}
+
+const increaseTemp = () => {
+    state.currentTemp += 1;
+    const temp = document.getElementById("temp_val");
+    temp.textContent = state.currentTemp;
+    updateGardenTemp(state.currentTemp)
+}
+
+const decreaseTemp = () => {
+    state.currentTemp -= 1;
+    const temp = document.getElementById("temp_val");
+    temp.textContent = state.currentTemp;
+    updateGardenTemp(state.currentTemp);
+}
```

Comment:
I like the use of `updateGardenTemp` as a helper function.  However you could combine these functions

```suggestion
const changeTemp = (increment) => {
    state.currentTemp += increment;
    const temp = document.getElementById("temp_val");
    temp.textContent = state.currentTemp;
    updateGardenTemp(state.currentTemp)
}
```

Then you can use an anonymous function to call it.

```javascript
  increaseBtn.addEventListener("click", () => changeTemp( +1 ));
```